src/test/java/seedu/duke/ActivityFindTest.java

```diff
@@ -0,0 +1,46 @@
+package seedu.duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ActivityFindTest {
+    void createObjects(ActivityList dummyList) {
+        dummyList.addActivity(new Food("Apple", 50));
+        dummyList.addActivity(new Food("Banana", 100));
+        dummyList.addActivity(new Food("Apple pie", 55));
+        dummyList.addActivity(new Exercise("Juggle Apples", 100));
+    }
+
+    @Test
+    void findActivityDescription_success() {
+        ActivityList dummyList = new ActivityList();
+        createObjects(dummyList);
+        String output = "";
+        for (int i = 0; i < dummyList.getNumberOfActivities(); i++) {
```

Comment:
You can use the methods in findDescriptionCommand class and assert using those methods.